### PR TITLE
Adds documentation for Twilio

### PR DESCRIPTION
**Pull request by seanbermejo**
For issue: https://github.com/worthwhile/django-herald/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +271,22 @@ And in your template you would refer to it like this, and you would not need to
...
+
+You can retrieve these values on [Twilio Console](https://twilio.com/console). Once you have retrieve the necessary ids, you can place those to your `settings.py`.
+
+For reference, Twilio has some great tutorials for python.
+[Twilio Python Tutorial](https://www.twilio.com/docs/sms/quickstart/python)
+
+```
+# Twilio configurations
+# values taken from `twilio console`
+TWILIO_ACCOUNT_SID = "ACe0bfc543af2cf98996beac0175b86c12"
```
Comment (jproffitt) [inline README.md:284]: @seanbermejo This isn't a real account SID is it?

Comment (seanbermejo) [reply README.md]: Yeah sorry about that. That's just a testing account though. I updated the docs.

**Comment by jproffitt** [comment]
Thanks!

**Comment by seanbermejo** [comment]
Btw, would it be better if we also add `twilio` to `setup.py` that we can install via `pip install django-herald[twilio]` ?

**Comment by jproffitt** [comment]
Sure that’s a good idea

**Pull request merged**